support-workers/src/test/scala/com/gu/acquisitions/BigQuerySpec.scala

```diff
@@ -38,17 +39,14 @@ class BigQuerySpec extends AsyncFlatSpec with Matchers with LazyLogging {
       .getService
 
   "BigQuery" should "be able to run a query" in {
+
+    val yesterday = ISODateTimeFormat.dateTime().print(DateTime.now().minusDays(1))
     val query =
-      s"""select * from ${AcquisitionEventTable.datasetName}.${AcquisitionEventTable.tableName} where amount = 9999 and event_timestamp > TIMESTAMP("2020-12-14 00:20:00");"""
```

Comment:
This test was selecting a massive number of records and then spamming the logs with them. I've updated it to just select records later than yesterday and not to log the values